.rubocop.yml

```diff
@@ -64,6 +64,8 @@ Naming/PredicateName:
 
 Style/ConditionalAssignment:
   EnforcedStyle: assign_inside_condition
+  SingleLineConditionsOnly: false
+  IncludeTernaryExpressions: false
```

Comment:
```suggestion
  IncludeTernaryExpressions: true
```

This will check for things like:

```ruby
# bad
condition ? answer = 'yes' : answer = 'no'
```

As it should.  

We would instead do:

```ruby
# good
answer = condition ? 'yes' : 'no'
```

Reply:
The enforced style I'm using is `assign_inside_condition` so we want to not enforce that within ternaries.

Sadly I can't choose to enforce one style in ternaries and one outside, so I'm choosing to keep the `assign_inside_condition` as the default.

Reply:
So you want to assign, unconditionally, inside conditional statements?  I am not sure I understand.  I understand the conditional assignment in a conditional branch, but not unconditionally.

In other words:

```ruby
# good
condition ? answer = 'yes' : 'condition is not true'
```

As an if statement, it is the case as well (no real difference between a ternary statement and an if statement).

Reply:
I want:

```
if foo
  bar = 1
else
  bar = 2
end

thing == true ? "true" : "false"
```

Which is what this enforces as far as I can tell :)

Reply:
(Updated the example for clarity)

Reply:
Yeah, and that is what is likely the thing that doesn't make sense to me.

```ruby
bar = if foo
           1
         else
           2
        end
```

Because unconditional things don't belong in conditional branches.

The way it is, I think you are right, it does enforce the unconditional assignment in conditional branches, while ignoring (not enforcing either way) the ternary statement use of that.

(Does it hurt anything?  My brain, but that doesn't matter.)